src/schema.rs

```diff
@@ -633,3 +633,39 @@ fn default_true() -> bool {
 fn default_false() -> bool {
     false
 }
+
+#[derive(serde_derive::Deserialize, Debug)]
+#[serde(deny_unknown_fields, rename_all = "kebab-case")]
+pub(crate) struct Repo {
+    pub org: String,
+    pub name: String,
+    pub description: String,
+    pub bots: Vec<String>, // TODO: should this be an enum?
```

Comment:
Yes, I think so (or we want some validation at least). Each bot will likely end up with slightly different privileges -- or even move to a github app that gets invited to particular repos, so that it's very "non-stringy".